lib/pbench/cli/agent/commands/results/results_options.py

```diff
@@ -15,6 +15,14 @@ def results_common_options(f):
             type=click.Choice(["public", "private"], case_sensitive=False),
             help="pbench tarball access permission",
         ),
+        click.option(
+            "-m",
+            "--metadata",
+            required=False,
+            default=[],
+            multiple=True,
+            help="metadata to add information for server",
```

Comment:
Do we want to expand a little more on this help? Maybe something like:
```
Optional flag to add metadata to the dataset, 
the server may take additional actions based on the value.
```
Should we add an example to this help like `server.archiveonly:y` would mark the dataset for archival only?